### PR TITLE
{bio}[intel/2021b] Bambi v0.7.1 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)
* https://github.com/easybuilders/easybuild-easyconfigs/pull/14810
* https://github.com/easybuilders/easybuild-easyconfigs/pull/14808

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Dependencies:

- https://github.com/easybuilders/easybuild-easyconfigs/pull/14810
- https://github.com/easybuilders/easybuild-easyconfigs/pull/14808

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3202.victini.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7ef47787ecbcf72ed8ddf6c354e34037 for a full test report.

File: easybuild/easyconfigs/b/Bambi/Bambi-0.7.1-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    ('SciPy-bundle', '2021.10'),
+    ('statsmodels', '0.13.1'),
+    ('ArviZ', '0.11.4'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('formulae', '0.2.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/b/Bambi/Bambi-0.7.1-intel-2021b.eb:25]: @ItIsI-Orient Hold on, Bambi itself is not actually being installed here?!

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/Bambi/Bambi-0.7.1-intel-2021b.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+to make it extremely easy to fit Bayesian mixed-effects models common in biology,
+social sciences and other disciplines."""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('statsmodels', '0.13.1'),
+    ('ArviZ', '0.11.4'),
```
Comment (boegel) [inline easybuild/easyconfigs/b/Bambi/Bambi-0.7.1-intel-2021b.eb:18]: @ItIsI-Orient Looks like a dependency is missing now? Test fails with:

```
bambi 0.7.1 requires pymc3, which is not installed
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3526.doduo.os - Linux UNKNOWN UNKNOWN, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/38d5e74e7f31c01facd33cc438b180bc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2603.swalot.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/1cac7ef94384d5d918ff84f104d357da for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**